impl/src/main/java/org/ehcache/config/builders/UserManagedCacheBuilder.java

```diff
@@ -80,10 +79,10 @@
 import static org.ehcache.config.ResourceType.Core.OFFHEAP;
 import static org.ehcache.config.builders.ResourcePoolsBuilder.newResourcePoolsBuilder;
 import static org.ehcache.core.internal.service.ServiceLocator.dependencySet;
+import static org.ehcache.core.internal.service.ServiceLocator.findSingletonAmongst;
```

Comment:
Could you make sure your IDE does not go wild in re-ordering imports when you are not touching them?